OrbitCore/ProcessUtils.cpp

```diff
@@ -259,23 +248,32 @@ bool ProcessList::Contains(uint32_t a_PID) const {
   return false;
 }
 
-//-----------------------------------------------------------------------------
 void ProcessList::SetRemote(bool value) {
-  for (std::shared_ptr<Process>& process : m_Processes) {
+  for (std::shared_ptr<Process>& process : processes_) {
     process->SetIsRemote(value);
   }
 }
 
-//-----------------------------------------------------------------------------
 std::shared_ptr<Process> ProcessList::GetProcess(uint32_t pid) const {
   std::shared_ptr<Process> result = nullptr;
-  auto iter = m_ProcessesMap.find(pid);
-  if (iter != m_ProcessesMap.end()) result = iter->second;
+  auto iter = processes_map_.find(pid);
+  if (iter != processes_map_.end()) result = iter->second;
   return result;
 }
 
-//-----------------------------------------------------------------------------
+const std::vector<std::shared_ptr<Process>>& ProcessList::GetProcesses() const {
+  return processes_;
+}
+
+size_t ProcessList::Size() const {
+  return processes_.size();
+}
+
+void ProcessList::AddProcess(std::shared_ptr<Process> process) {
+  processes_.push_back(process);
```

Comment:
Is it necessary to update processes_map_ here as well?

Reply:
interesting... I would say yes, even though it worked without it before. Will fix, also probably refactor Refresh to use this function instead of doing all the work by itself.

Reply:
done